config/containers/logging/configure.md

```diff
@@ -43,32 +42,30 @@ example sets two configurable options on the `json-file` logging driver:
 {
   "log-driver": "json-file",
   "log-opts": {
-    "max-size": "10m",
+    "max-size": "10m",		
```

Comment:
is that a trailing tab? If so, please remove it

Reply:
Got it.